bonus_guides/Y_generate_project_docs.md

```diff
@@ -0,0 +1,52 @@
+The correct invocation of helper modules and functions can be intimidating because
```

Comment:
I would start here with an overarching paragraph on adding docs generation to your project. and then get into how it's ALSO helpful when using macro generated functions like *_path